bindings/java/fdbJNI.cpp

```diff
@@ -1042,8 +1036,43 @@ JNIEXPORT void JNICALL Java_com_apple_foundationdb_FDB_Network_1stop(JNIEnv *jen
 }
 
 jint JNI_OnLoad(JavaVM *vm, void *reserved) {
+    JNIEnv *env;
```

Comment:
Some of the lines are using spaces instead of tabs.

Reply:
Thanks will fix...